src/components/StudyRoom/StudyRoomHome.js

```diff
@@ -12,33 +12,52 @@ import Card from "@mui/material/Card";
 import axios from "axios";
 
 export default function StudyRoomHome() {
-    useEffect(() => {
-        if (localStorage.getItem("email")) {
-            if (localStorage.getItem("email") === "")
-                window.location = "/calendar"
-        }
-    }, [])
+    const [errorMessage, setErrorMessage] = useState('')
+    const [roomData, setRoomData] = React.useState({
+        title:'',
+        color: '',
+        description:'',
+        avatarText:'',
+        createdOn:'',
+    });
 
     //todo get room title,icon,description and creation date in the study room snippet
+    useEffect(() => {
+        getData();
+    },[])
+
+    function getData(){
+        let user = localStorage.getItem("username");
```

Comment:
users are identified by email, not username

Reply:
not in the scope of this PR